### ISSUE TITLE
Add keyboard focus support to show cell tooltips

### ISSUE THREAD

**Issue by marciogurka** [bryntum/support#5539]
[Forum post](https://forum.bryntum.com/viewtopic.php?f=51&t=22880&p=113100#p113100)

Hi,

We have found that the cell tooltips are not accessible for users who are using screen readers or for users who are solely using keyboard for navigation.

The W3C guidelines are a bit vague, but they do suggest:

> A tooltip is a popup that displays information related to an element when the element receives keyboard focus or the mouse hovers over it. It typically appears after a small delay and disappears when Escape is pressed or on mouse out.

[https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/](https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/)

Would it be possible to include an option to show cell tooltips when the cell has focus, instead of only when the mouse moves over the cell?

**Comment by marciogurka** [comment]
[+1](https://forum.bryntum.com/viewtopic.php?p=140976#p140976) for this

**Comment by ghulamghousdev** [comment]
Another client reported this issue: https://forum.bryntum.com/viewtopic.php?p=145926#p145926
